api/crossmodel/client_test.go

```diff
@@ -91,3 +93,123 @@ func (s *crossmodelMockSuite) TestOfferFacadeCallError(c *gc.C) {
 	c.Assert(errors.Cause(err), gc.ErrorMatches, msg)
 	c.Assert(results, gc.IsNil)
 }
+
+func (s *crossmodelMockSuite) TestShow(c *gc.C) {
+	url := "local:/u/fred/prod/db2"
+
+	desc := "IBM DB2 Express Server Edition is an entry level database system"
+	endpoints := []params.RemoteEndpoint{
+		params.RemoteEndpoint{"db2", "mysql", "provider"},
+		params.RemoteEndpoint{"log", "http", "requirer"},
+	}
+	serviceTag := "service-hosted-db2"
+
+	called := false
+
+	apiCaller := basetesting.APICallerFunc(
+		func(objType string,
+			version int,
+			id, request string,
+			a, result interface{},
+		) error {
+			called = true
+
+			c.Check(objType, gc.Equals, "CrossModelRelations")
+			c.Check(id, gc.Equals, "")
+			c.Check(request, gc.Equals, "Show")
+
+			args, ok := a.([]string)
+			c.Assert(ok, jc.IsTrue)
+			c.Assert(args, gc.DeepEquals, []string{url})
+
+			if points, ok := result.(*params.RemoteServiceInfos); ok {
```

Comment:
Oh yes - it's an amazing pickup on your behalf and an awful oversight on mine!
Thank you :D